### PR TITLE
[ERE-1743] Vendor django useraudit

### PR DESCRIPTION
**Pull request by ScriptSmith**

### CODE REVIEW

File: rdrf/useraudit/test_settings.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+# List of finder classes that know how to find static files in
+# various locations.
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+)
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = '7h(an4%q1ycpr_%18p2tmc#_@-qrp9nn8=m_w+f0(!+kjb!!ok'
```
Comment (ppettitau) [inline rdrf/useraudit/test_settings.py]: I know this is `test_settings` but I just thought I'd ask the question anyway, Is this `SECRET_KEY` value ok to be here in plain text?

File: rdrf/useraudit/test_urls.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    from django.urls import include, path
+    PRE_DJANGO_2 = False
+except ImportError:
+    from django.conf.urls import include, url
+    PRE_DJANGO_2 = True
+
+
+admin.autodiscover()
+
+if PRE_DJANGO_2:
```
Comment (ppettitau) [inline rdrf/useraudit/test_urls.py]: Do we need this `PRE_DJANGO_2` logic?

File: docker/dev/docker-entrypoint.sh [hunk 3]
```diff
@@ -239,7 +239,7 @@ if [ "$1" = 'runtests' ]; then
     export DJANGO_SETTINGS_MODULE="${DJANGO_SETTINGS_MODULE}"_test
 
     set -x
-    args="rdrf/rdrf/testing/unit rdrf/report/tests"
+    args="rdrf/rdrf/testing/unit rdrf/report/tests rdrf/useraudit/tests"
```
Comment (ppettitau) [inline docker/dev/docker-entrypoint.sh:242]: Need to include `rdrf/useraudit/tests` below in `runtests_coverage`.

**Review by ppettitau — CHANGES_REQUESTED** [review]

File: rdrf/useraudit/test_urls.py (hunk 2 above)
Comment (ScriptSmith) [reply rdrf/useraudit/test_urls.py]: Nope, removed those files & middleware + logic related to django 1 support

File: docker/dev/docker-entrypoint.sh (hunk 3 above)
Comment (ScriptSmith) [reply docker/dev/docker-entrypoint.sh:242]: Fixed

Comment (ppettitau) [reply docker/dev/docker-entrypoint.sh:242]: I can't see the changes for some reason. 
Also the CI Checks are still not executing the useraudit unit tests:
```
  [ OK ] Mon Jun 19 06:11:58 UTC 2023 - connected to dbtest 5432
+ args='rdrf/rdrf/testing/unit rdrf/report/tests'
+ '[' '' '!=' '' ']'
+ cd /app
+ coverage run -m pytest rdrf/rdrf/testing/unit rdrf/report/tests
```

Comment (ScriptSmith) [reply docker/dev/docker-entrypoint.sh:242]: My bad, must've missed that.

**Review by ppettitau — APPROVED** [review]

**Pull request merged**